src/connect/components/folder-item/folder-item.tsx

```diff
@@ -0,0 +1,45 @@
+import { DivProps, Icon } from '@/powerhouse';
+import React from 'react';
+import { twMerge } from 'tailwind-merge';
+
+export interface FolderItemProps extends DivProps {
+    title: string;
+    onOptionsClick?: DivProps['onClick'];
+}
+
+export const FolderItem: React.FC<FolderItemProps> = ({
+    title,
+    onOptionsClick,
+    ...divProps
+}) => {
+    const containerStyles = twMerge(
```

Comment:
no need to change but just thought I'd say to potentially save you some future effort — you don't need to do `flex-row` here because that's already the default. Just doing `flex` is all you need to get this result

Reply:
nice catch! fix here: [3f6f985](https://github.com/powerhouse-inc/design-system/pull/128/commits/3f6f9850fc217071a42a1280adaa6cc0bc30069c)